wordpress/Dockerfile

```diff
@@ -2,7 +2,7 @@ ARG WORDPRESS_IMAGE_TAG=5-php7.3-apache
 FROM wordpress:${WORDPRESS_IMAGE_TAG} AS release
 
 # Add sudo in order to run wp-cli as the www-data user
-RUN apt-get update && apt-get install -y sudo less
+RUN apt-get update -y && apt-get install -y sudo less
```

Comment:
apt-get is fuc***g loud. It can be silenced a bit by using this: `apt-get -qqqq .... 1>/dev/null`